File: ControlRemote/Web/Dto/RegisterModel.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class RegisterModel
    {
        public string Login { get; set; }

        [DataType(DataType.Password)]
        public string Password { get; set; }

        [DataType(DataType.Password)]
        public string ConfirmPassword { get; set; }
    }
}

File: ControlRemote/Web/ClientApp/ControlRemoteApp/src/app/dto/RegisterModel.ts
export class RegisterModel {
    login: string;
    password: string;
    confirmPassword: string;

    public constructor(_login: string, _password: string, _confirmPassword: string) {
        this.login = _login;
        this.password = _password;
        this.confirmPassword = _confirmPassword;
    }
}